components/kyma-environment-broker/internal/orchestration/manager/manager.go

```diff
@@ -121,6 +124,13 @@ func (m *orchestrationManager) resolveOperations(o *internal.Orchestration, poli
 			return result, errors.Wrap(err, "while resolving targets")
 		}
 
+		if o.Parameters.Kyma.Version == "" {
```

Comment:
One tiny thing, and the PR can be merged:

These paramaters sets should go after the `for _, r := range runtimes {` loop below, because in upgrade kyma factory's `NewOperation()` there is a code which checks if an input version was provided for the orchestration, and calculates oper runtime target version accordingly:
https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/orchestration/manager/upgrade_kyma.go#L74

We must not trigger this code accidentally with the default kyma version.